bin/configs/kotlin-jvm-jackson.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: kotlin
+outputDir: samples/client/petstore/kotlin-jvm-jackson
+inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore.yaml
```

Comment:
I did a test locally but got the following errors:
```
➜  kotlin-jvm-jackson git:(fix_retrofit_generator_with_jackson) ✗ gradle build
Starting a Gradle Daemon (subsequent builds will be faster)

> Task :compileKotlin
e: /Users/williamcheng/gitclone/yonatankarp_fix_retrofit_generator_with_jackson/samples/client/petstore/kotlin-jvm-jackson/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt: (60, 28): Unresolved reference: Builder
e: /Users/williamcheng/gitclone/yonatankarp_fix_retrofit_generator_with_jackson/samples/client/petstore/kotlin-jvm-jackson/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt: (60, 49): Unresolved reference: Builder
e: /Users/williamcheng/gitclone/yonatankarp_fix_retrofit_generator_with_jackson/samples/client/petstore/kotlin-jvm-jackson/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt: (75, 28): Unresolved reference: Builder
e: /Users/williamcheng/gitclone/yonatankarp_fix_retrofit_generator_with_jackson/samples/client/petstore/kotlin-jvm-jackson/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt: (75, 49): Unresolved reference: Builder

> Task :compileKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

```

Can you please take a look when you've time?

Also update use 3.0 spec instead: `modules/openapi-generator/src/test/resources/3_0/petstore.yaml`

Reply:
Please commit the new sample files as well so that github workflow will trigger the build.

Reply:
There was indeed a but with my fix - my bad! 🤦‍♂️ 
I have fixed it and updated the sample app and the spec version however seems that `GitHub Actions` isn't picking the changes yet, and `CircleCI` still failing after I have rebased on `master`